release-notes/README.md

```diff
@@ -4,755 +4,19 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
-
-The included features with this release are:
-
-* Ability to create XDM Experience Events and send them to Experience Edge Network. An optional Experience Edge response callback can be registered per event.
-* The XDM Experience Events can be sent to individual Adobe Experience Platform datasets when a custom dataset identifier is set at event level.
-* Events persistence.
-* ECID is automatically attached on each XDM Experience Event request.
-* Integration with AEPAssurance extension enabling new insights in Project Griffon about the XDM Experience Event processing and XDM data validation for an improved validation and debugging experience.
-* Detailed warning/error messages are available through Project Griffon UI and logs.
-
-This extension library is available as an open sourced project \([adobe/aepsdk-edge-ios](https://github.com/adobe/aepsdk-edge-ios)\) on GitHub.
-
-These iOS libraries, written in Swift, are available for integration via SPM, XCFramework and Cocoapods and can be used in Swift and Objective-C applications.
-
-### Android Edge 1.0.0
-
-The Adobe Experience Platform Edge Network \(Edge\) mobile extension is now available in Android! This extension allows for sending XDM formatted data to Adobe Experience Platform and Adobe Experience Cloud solutions, by leveraging Experience Edge capabilities.
-
-The included features with this release are:
-
-* Ability to create XDM Experience Events and send them to Experience Edge Network. An optional Experience Edge response callback can be registered per event.
-* The XDM Experience Events can be sent to individual Adobe Experience Platform datasets when a custom dataset identifier is set at event level.
-* ECID is automatically attached on each XDM Experience Event request.
-* Integration with Assurance extension enabling new insights in Project Griffon about the XDM Experience Event processing and XDM data validation for an improved validation and debugging experience.
-* Detailed warning/error messages are available through Project Griffon UI and logs.
-
-This SDK is compatible with Android Core 1.5.7 and above.
-
-### Adobe Experience Platform Edge Network Launch Extension 1.0.2
+Upgrade the following flutter package to reference Android libraries from mavenCentral :
 
-* Selector for the Edge Configuration.
-* AEP Request Event with optional XDM event type is available for Launch rules.
+* [Flutter ACPCore v2.0.1](https://pub.dev/packages/flutter_acpcore/versions/2.0.1)
+* [Flutter ACPUserProfile v2.0.1](https://pub.dev/packages/flutter_acpuserprofile/versions/2.0.1)
+* [Flutter ACPPlaces v2.0.1](https://pub.dev/packages/flutter_acpplaces/versions/2.0.1)
+* [Flutter ACPPlacesMonitor v2.0.1](https://pub.dev/packages/flutter_acpplaces_monitor/versions/2.0.1)
 
-## January 20, 2021
+### Flutter Assurance 2.0.1
 
-### iOS Audience 2.3.0
+- Migrate Android build from jcenter to mavenCentral
```

Comment:
Should we use a similar wording here:
Updated ACPAssurance package to reference the Android library from Maven Central repository.